packages/logger/src/keys.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export namespace BindingKeys {
+  export namespace System {
+    export const LOGGER_PROVIDER: string = 'system.logger_provider';
```

Comment:
Let's use the component name as the prefix, i.e. `logger.`. If we end up exporting a sequence action as I mentioned in one of my comments above, then I think this should become:

```ts
export namespace Logger {
  export namespace SequenceActions {
    export const SETUP_LOGGER  = 'logger.sequence.actions.setupLogger';
  }
}
```